main/getting-started/alpha.md

```diff
@@ -0,0 +1,60 @@
+# Agoric Alpha	
+Release Version: Adriatic
+
+![Alpha Cover](./assets/alpha-cover.png)
+
+We’re building components to power the DeFi ecosystem. Come take a peek under the hood and help us scale our next-gen infrastructure. The Alpha Adriatic release provides you access to our toolset, API, and public chain.
+
+[Start Coding](https://agoric.com/documentation/getting-started/)
+[Join Community](https://discord.gg/gC9z6US)
+
+---------------------
+
+## What you can build in Alpha Adriatic 
+
+Numerous transaction components -- what we think of as DeFi Legos -- are available in the Alpha Adriatic release. We’ll be adding to the pool with our growing community’s help and participation in the coming months as we scale contributions. The initial slate of elements includes the following: 
```

Comment:
I know I saw a discussion of the trademark issues fly by, but I think capitalizing "Legos" is asking for it. As it's text and not a title, capitalization means it's a proper noun; i.e. the Danish toy brick. If we're going to claim generic use, it should be "legos"; i.e. you don't say you'll make a Xerox, but that you'll make a xerox if you want a generic use claim. 

I'd redo the last two sentences as "In the coming months, with your and the rest of our community's help, we'll be adding more to this initial list: